subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1143,7 +1143,7 @@ static int cmd_wifi_reg_domain(const struct shell *sh, size_t argc,
 {
 	struct net_if *iface = net_if_get_first_wifi();
 	struct wifi_reg_domain regd = {0};
-	int ret;
+	int ret, i = 0;
```

Comment:
`chan = 0`?

Reply:
Keeping a name as "chan" which will be used as "index" will be confusing . chan = 0, 1, 2  
"I"or "chan_idx" will be better.

Reply:
sure, some index referring `chan` than just generic `i`